nodejs/user-interface/modals/new-task.js

```diff
@@ -22,21 +20,11 @@ module.exports = (prefilledTitle) => {
       Blocks.Input({ label: 'New task', blockId: 'taskTitle' }).element(
         textInput(prefilledTitle),
       ),
-      Blocks.Input({ label: 'Assign user', blockId: 'taskAssignUser', optional: true }).element(
+      Blocks.Input({ label: 'Assign user', blockId: 'taskAssignUser' }).element(
```

Comment:
If we have a default value (the creating user), then there's no need to make this optional, since there'll always be something pre-filled